### DISCUSSION TITLE
Meeting 5: May 30th, 14:00 UTC

### DISCUSSION THREAD

**Discussion by ml-evs** [marda-alliance/metadata_extractors#19] [Announcements]
# 5th MaRDA Extractors WG meeting

When: [30th May, 14:00 UTC](https://www.timeanddate.com/worldclock/fixedtime.html?iso=20230530T16&p1=37)
Where: https://meet.jit.si/marda_extractors

Agenda:

TBD

**Comment by PeterKraus** [comment]
[Minutes from this meeting are up!](https://github.com/marda-alliance/metadata_extractors/blob/main/meetings/5-meeting-2023-05-30/minutes.md)

Thanks again to those who attended yesterday.
